docs/tutorials/1.5/run.md

```diff
@@ -0,0 +1,19 @@
+This is a small tutorial on how to run a HORNET node.
+
+**Note: This tutorial assumes that you have prebuilt HORNET. Learn [how to build HORNET here](./build.md)**
+
+
+The HORNET RPM package is ntot realesed for Chrysalis yet.
```

Comment:
`ntot realesed` => `not released`

And the more important package is the DEB package.... you should also mention that.
But maybe talk about "binaries are not released yet", because it also includes windows etc